plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changeslist/ChangesListViewImpl.ui.xml

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
What the reason add ui.xml only for bind one FlowPanel?

Reply:
I moved it to ui.xml to make our code more identical. But if you think that it has some drawbacks, it isn't problem to revert.